### DISCUSSION TITLE
Is anyone has experience deploying a self-hosted instance on Azure?

### DISCUSSION THREAD

**Discussion by fslef** [outline/outline#2655] [Self Hosting]
Hello community,

Is anyone has experience deploying a self-hosted instance of outline on Azure?
Is there any good pointer or a step-by-step guide that would help for the deployment?

Thanks a lot in advance,

**Comment by scott-dunt** [comment]
Sorry to hear you're running on Azure ;-)  I will offer my notes on setup self hosted on AWS: https://github.com/outline/outline/discussions/2383
